scripts/SANS/sans/user_file/user_file_parser.py

```diff
@@ -2085,11 +2085,16 @@ def __init__(self):
 
```

Comment:
Should we remove the 
```
self._print = "\\s*PRINT\\s+"
self._print_pattern = re.compile(start_string + self._print + "\\s*.*\\s*" + end_string)
```
lines above?

Reply:
Yes I think so have now done. I have left in the static member functions below because I think they are used elsewhere when deciding which parser to use and removing them currently breaks the unit tests

Reply:
That sounds fine. It's good that there are tests that cover this and thanks for adding the new ones.